rtb/piano_di_progetto_ver0.2.0.typ

```diff
@@ -229,3 +236,75 @@ Gli analisti evidenziano le difficoltà e i dubbi relativa alla comprensione dei
 I verificatori suggeriscono infine di incaricare più di una persona alla verifica dei documenti ad alto contenuto, quali possono essere le Norme di Progetto e il Piano di Progetto, al fine di garantire una verifica ottimale.
 
 
+#pagebreak()
+
+=== Sprint 2
+- *Inizio*: 28/11/2024
+- *Fine*: 13/12/2024
+- *Rischi attesi*: #link(<RO-2>)[RO-2], #link(<RO-3>)[RO-3], #link(<RT-1>)[RT-1]
+
+==== Attività dello sprint
+//Quello che è stato fatto o quello che è da fare ?
+Le attività che il _team_ si è prefissato di portare a termine durante la durata del secondo _sprint_ sono le seguenti:
+- Redazione verbale esterno ed interno del 27/11/2024.
+- Creare una prima versione del simulatore dati.
+  - È stato deciso di dedicare la prima settimana dello _spint_ allo studio ed all'approfondimento delle tecnologie: #rifGlossario("Docker"), #rifGlossario("Apache Kafka") ed #rifGlossario("Apache Nifi")
+  - Mentre di dedicare la seconda settimana all'implementazione concreta del simulatore, utilizzando le tecnologie studiate.
+- Avanzamento nel documento Analisi dei Requisiti, in particolare: 
+  - Migliorando i vecchi casi d'uso dopo aver risolto alcuni dubbi in seguito al confronto con l'azienda.
+  - Scrivendo nuovi casi d'uso.
+- Continuazione stesura Piano di Progetto, in particolare:
+  - Scrittura retrospettiva primo _sprint_.
+  - Scrittura pianificazione secondo _sprint_.
+- Inizio stesura del Piano di Qualifica.
+- Modifica alle Norme di Progetto, per via del nuovo aggiornamento riguardo al sistema di _versioning_ (spiegazione completa nel VI_2024-11-27).
+- Modifica delle versioni dei seguenti documenti, per coerenza con il nuovo sistema di versionamento:
+  - Glossario
+  - Analisi dei Requisiti
+  - Piano di Progetto
+  - Norme di Progetto
+- Modifica al _template_ ed al _test_ del Glossario.
+  - La modifica al _template_ vedrà l'implementazione della generazione automatica dei grafici e dell'aggiunta automatica dell'indice delle immagini, e delle tabelle, al _template_.
```

Comment:
- si ripete "template" sia all'inizio che alla fine della frase, inoltre specificherei i grafici, immagini e tabelle cosa riguardano e in quali documenti
- riformulerei la frase per evitare le 3 congiunzioni